developers/weaviate/client-libraries/typescript/typescript-v3.md

```diff
@@ -127,222 +129,157 @@ console.log(client)
 </TabItem>
 </Tabs>
 
-:::tip Troubleshooting imports with TypeScript
-If you are having any issues with the import statement in TypeScript (e.g. if `weaviate` is `undefined`), try adding `"esModuleInterop": true` to your `"compilerOptions"` in `tsconfig.json`.
-:::
-
-### Type definitions
-
-The type definitions can be found under each bundles respective folder; the subdirectory of `node/cjs` and `node/esm` in the `*.d.ts` files, for example as shown on the [npm package page](https://www.npmjs.com/package/weaviate-client/v/3.0.0-beta.17?activeTab=code).
+## Usage
 
-// add code example 
-
-## Authentication
-
-import ClientAuthIntro from '/developers/weaviate/client-libraries/_components/client.auth.introduction.mdx'
+### Design philosophy
 
-<ClientAuthIntro clientName="TypeScript"/>
+The `v3` client interacts with collections as the primary way to work with objects in your Weaviate database.
 
-### WCS authentication
+Your application code creates a connection to a collection. Then, the code defines data operations on that collection. This example returns objects from the JeopardyQuestion collection.
```

Comment:
I don't think this is true (unless the TS client works differently).

The application code creates an object that represents a collection, then operations such as CRUD or search operations are performed against that object. The client afaik does not create any type of persistent connection against a collection.

Reply:
Maybe there's a confusion of terms? 

From the user's point of view, there is an object that represents the collection, but that object has to connect to the dbase in order to do anything. 

I think the object is created when users run one of the connect_to methods to create the network connection. Am I missing a step?

Reply:
fixed this!